src/js/components/RangeSelector/EdgeControl.js

```diff
@@ -29,7 +29,23 @@ const EdgeControl = forwardRef(
     const { theme } = useThemeValue();
     const [focus, setFocus] = useState(false);
     const { cursor, fill } = DIRECTION_PROPS[direction];
-    const size = parseMetricToNum(theme.global.spacing) / 2;
+    const themeEdgeSize = theme.rangeSelector?.edge?.size;
+    let size;
+    if (themeEdgeSize != null) {
```

Comment:
Any reason this is != rather than !==? Or should it simply be `if (themeEdgeSize)`? According to the type it is supposed to always be a tshirt size or some other string (probably non-empty or 0). So it seems like `if (themeEdgeSize)` should be good. If you need to support 0 it should still be something more robust (it seems like this may be possible given how you default to using `themeEdgeSize` if there's not a mapping in `global.edgeSize`.)

Reply:
agree it can be simplified